### ISSUE TITLE
Text prompt embedding size

### ISSUE THREAD

**Issue by surmenok** [facebookresearch/segment-anything#19]
Thank you for releasing the model.
The paper mentions that text prompts are encoded using a pretrained `ViT-L/14@336px` CLIP model. CLIP embedding from this model are of size 768 while SegmentAnything prompt embedding is of size 256. Are there any extra steps for converting CLIP embeddings before feeding into the model?

**Comment by Ankur-singh** [comment]
Yes, if you go through the code (in particular `prompt_encoder.py`) you will see that only point and bbox embeddings are supported in Sparse embedding. 

Any plans of releasing text prompt support in the near future?

**Comment by aniki-ly** [comment]
To properly implement SVM into vision systems, having text prompt support is essential. I was wondering if there are any plans in the near future to release support for text prompts?
